ansible/roles/discourse/templates/standalone.yml.j2

```diff
@@ -97,5 +97,8 @@ run:
   - exec: echo "Beginning of custom commands"
   ## If you want to set the 'From' email address for your first registration, uncomment and change:
   ## After getting the first signup email, re-comment the line. It only needs to run once.
-  #- exec: rails r "SiteSetting.notification_email='{{ dis_smtp_user }}'"
+{% if discourse_container.exists %}
+{% else %}
+  - exec: rails r "SiteSetting.notification_email='{{ dis_smtp_user }}'"
```

Comment:
I don't understand what is the issue with this sending email notifications. We use a testing email

Reply:
This conditional check if Discourse exists and enable the first signup email setting.
If Discourse container exists then there is no need for the first signup email as admin users have been already setup.

Reply:
Well, what if we want to test a different email formatting? I think it's a good practice to allow the same functionality as the full server, especially because we are not sharing the same mailing account... So I'd say we should remove this

Reply:
> Well, what if we want to test a different email formatting? 

What do you mean by a different email formatting?

The option `SiteSetting.notification_email` is required only once upon the initial installation of the Discourse instance. Then this option should not be used accord to Discourse config comment.

Reply:
Never mind. I've got it now.. You can merge it